app/api/post/route.ts

```diff
@@ -0,0 +1,24 @@
+import { auth } from "@/auth"
+import { NextResponse } from "next/server"
+import { prisma } from '@/lib/createPost'
+
+
+export const POST = auth(async (req) => {
+    const data = await req.json()
+    if (req.auth) {
+        await prisma.post.createPost({
+            title: data.title,
+            latitude: data.latitude,
+            longitude: data.longitude,
+            ownerId: data.ownerId,
+            description: data.description,
+            availableSpots: data.availabeSpots,
+            name: data.name,
+            email: data.email
+        })
+        return NextResponse.json({ message: 'Authenticated'}, {status: 201})
```

Comment:
I think 'Authenticated' as a response message is misleading for an endpoint that creates a post. Return something like 'Created'.